util/dvsim/SimCfg.py

```diff
@@ -528,8 +528,7 @@ def gen_results_sub(items, results, fail_msgs):
     def gen_results_summary(self):
 
         # sim summary result has 5 columns from each SimCfg.results_summary
-        header = ["Name", "Passing", "Total", "Pass Rate"]
```

Comment:
This got inadvertently introduced by #1794. 
The correct fix is on line 507 and not here. Replace:
```python
            else:
                self.results_summary["Coverage"] = "--"
```

with:
```python
            elif self.cov:
                self.results_summary["Coverage"] = "--"
```